syncmaster/schemas/v1/transfers/file_format.py

```diff
@@ -47,11 +93,14 @@ class XML(BaseModel):
     type: XML_FORMAT
     root_tag: str
     row_tag: str
+    compression: XMLCompression | None = None
 
 
 class ORC(BaseModel):
     type: ORC_FORMAT
+    compression: ORCCompression = CSVCompression.NONE
```

Comment:
Let's change default values to following ones:
* `zlib` for ORC
* `snappy` for Parquet
* `gzip` for text-based formats (CSV, JSON, JSONLine)

If we can save storage space using sane defaults, let's do it.